### PR TITLE
Making Rockstar more Rockstarry by following PEP8

### PR DESCRIPTION
**Pull request by tusharmakkar08**
Did following changes: 
- Added `staticmethod` decorator
- Constants name capitalized
- Default initialization in `__init__.py`
- Making default argument immutable

### CODE REVIEW

File: rockstar/RockStar.py [hunk 1]
```diff
@@ -10,21 +10,22 @@
...
+DEFAULT_FILE_NAME = 'main.cpp'
 
 
 class RockStar:
 
-    def __init__(self, days=400, days_off=[], file_name=default_file_name,
-                 code=hello_world_c, off_fraction=0.0):
+    def __init__(self, days=400, days_off=(), file_name=DEFAULT_FILE_NAME,
+                 code=HELLO_WORLD_C, off_fraction=0.0):
+        self.repo = None
```
Comment (avinassh) [inline rockstar/RockStar.py:28]: whats the reason for this?

Comment (tusharmakkar08) [inline rockstar/RockStar.py:28]: It's better to define all the instance attributes within `__init__` for improving readability. Please refer to [this](http://stackoverflow.com/questions/19284857/instance-attribute-attribute-name-defined-outside-init) link.

Comment (avinassh) [inline rockstar/RockStar.py:28]: Great 👍

**Comment by avinassh** [comment]
Merging, thanks! 👍

**Comment by tusharmakkar08** [comment]
Cool :)

**Pull request merged**